tests/kernel/usb_drive.pm

```diff
@@ -49,8 +54,34 @@ sub run {
     assert_script_run "mount -t btrfs $device $mountpoint";
     assert_script_run "cd $mountpoint; md5sum -c $md5; cd /";
 
-    # cleanup
     assert_script_run("umount $mountpoint");
+
+    if (is_sle('16+')) {
+        $mountpoint = '~/mount';
+        $file = '~/file2';
+        $md5 = '~/md5';
+        $file_copy = "$mountpoint/file2";
+
+        assert_script_run "useradd -m -G users,disk unpriv -p $testapi::password";
+        install_package 'lklfuse';
+
+        assert_script_run "su unpriv";
+
+        assert_script_run "dd if=/dev/urandom of=$file bs=1M count=16";
+        assert_script_run "md5sum $file > $md5";
+
+        assert_script_run "mkdir -p $mountpoint";
+        assert_script_run "lklfuse -o type=btrfs $device $mountpoint";
+
+        assert_script_run "cp $file $file_copy";
+
```

Comment:
the blank lines in between seem arbitrary. I suggest to simply leave them all out